runtime/compiler/p/env/J9CPU.hpp

```diff
@@ -53,16 +53,18 @@ class OMR_EXTENSIBLE CPU : public J9::CPU
    CPU(const OMRProcessorDesc& processorDescription) : J9::CPU(processorDescription) {}
 
 public:
-   bool getPPCSupportsVMX();
-   bool getPPCSupportsVSX();
-   bool getPPCSupportsAES();
-   bool getPPCSupportsTM();
 
-   bool hasPopulationCountInstruction();
-   bool supportsDecimalFloatingPoint();
+   bool supportsFeature(uint32_t feature);
+   bool supports_feature_test(uint32_t feature);
 
    bool isCompatible(const OMRProcessorDesc& processorDescription);
    OMRProcessorDesc getProcessorDescription();
+
+   bool is(OMRProcessorArchitecture p);
+   bool isAtLeast(OMRProcessorArchitecture p);
+   bool isAtMost(OMRProcessorArchitecture p);
+
+   bool getPPCSupportsVSX();
```

Comment:
Why did we delete all other similar style methods but left this one?

Reply:
There are four instances of `getPPCSupportsVSX()` in closedj9. To make sure I don't break the build I'm temporarily keeping this in until we switch closedj9 to use the new APIs.